tbd.schema.json

```diff
@@ -267,15 +267,15 @@
           "type": "number"
         },
         "corner": {
-          "title": "Any wall corner [0°,135°) or (225°,360°] PSI",
+          "title": "Any wall corner [0deg,135deg) or (225deg,360deg] PSI",
```

Comment:
Linked to this [comment](https://github.com/rd2/tbd/pull/120#discussion_r1850927409). Fix works for both SDK v3.8.0 & v3.9.0.

More fixing to do ...